### PR TITLE
G2 2023 v6 issue#13790

### PR DESCRIPTION
**Pull request by b21ricni**
Documantation for codeviewerService Test

### CODE REVIEW

**Review by b21kurar — CHANGES_REQUESTED** [review]
Some minor changes needed, good work!

File: backend-models/service-tests/codeviewerService-test.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+Send{
+  $opt = "DELEXAMPLE";
+  $exampleId = 9023;
+}
+
+```
+
+###Service output
+```
+output{
```
[the remark this answers is not in the document]
Comment (b21ricni) [reply backend-models/service-tests/codeviewerService-test.md:368]: I understood that all output should be empty in the documentation

**Review by b21kurar — CHANGES_REQUESTED** [review]
do the output thing plz

**Review by b21kurar — APPROVED** [review]

**Comment by d21erisu** [comment]
nice

**Pull request merged**